go.mod

```diff
@@ -16,12 +16,12 @@ require (
 	github.com/spf13/afero v1.2.2
 	github.com/stretchr/testify v1.6.1 // indirect
 	github.com/xeipuuv/gojsonschema v1.2.0
-	golang.org/x/net v0.0.0-20200324143707-d3edc9973b7e // indirect
 	golang.org/x/sys v0.0.0-20200509044756-6aff5f38e54f // indirect
+	gonum.org/v1/netlib v0.0.0-20190331212654-76723241ea4e // indirect
 	gopkg.in/yaml.v2 v2.3.0 // indirect
-	k8s.io/api v0.18.2
-	k8s.io/apimachinery v0.18.2
+	k8s.io/api v0.18.6
+	k8s.io/apimachinery v0.18.6
```

Comment:
can we downgrade `k8s.io/api` and `k8s.io/apimachinery` back to `v0.18.2`? to be consist with api/odo/console/devworkspace?

Reply:
I am not controlling the version, this is automatically updated because devfile/api is using `v0.18.6`